src/Application/Filters/Actions/ActionWatershedFilter.cc

```diff
@@ -359,8 +347,8 @@ class WatershedFilterAlgo : public ITKFilter
 	virtual std::string get_layer_prefix() const
```

Comment:
Can I keep it as a selectable option?  I find it useful for quickly comparing the watersheds.